### DISCUSSION TITLE
Today's date as ISO

### DISCUSSION THREAD

**Discussion by SimplGy** [johnlindquist/kit#1117] [Share]
Completely ripped from https://github.com/johnlindquist/kit/discussions/1116 (Thanks Daniel!) I just like the ISO format better.

[Open today-timestamp in Script Kit](https://scriptkit.com/api/new?name=today-timestamp&url=https://gist.githubusercontent.com/SimplGy/2676c4ccad30d9c71f7096422eb5fd44/raw/0da283bff42524450ee9f162c24451e1a8332a47/today-timestamp.ts")

```js
// Name: today-timestamp
// Description: inserts today's date in "ISO" format 2023-02-11
// Snippet:

import '@johnlindquist/kit';

function twoDigits(number: number): string {
	return number.toString().padStart(2, '0');
}

await hide();
const today = new Date();
const formatted = `${today.getFullYear()}-${twoDigits(today.getMonth() + 1)}-${twoDigits(today.getDate())}`;
await keyboard.type(formatted);

```

**Comment by danielo515** [comment]
I now have both as part of my scripts 😄
